client/src/pages/RoutingPage/RoutingPage.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
change `About.jsx` for `RoutingPage.jsx`

might change to `AboutRoutingPage.jsx` to avoid ambiguity...?